packages/@webex/plugin-meetings/src/member/util.ts

```diff
@@ -49,6 +50,20 @@ MemberUtil.canReclaimHost = (participant) => {
 MemberUtil.getControlsRoles = (participant: ParticipantWithRoles): Array<ServerRoleShape> =>
   participant?.controls?.role?.roles;
 
+/**
+ * Checks if the participant has the brb status enabled.
+ *
+ * @param {ParticipantWithBrb} participant - the locus participant
```

Comment:
nit:

```suggestion
 * @param {ParticipantWithBrb} participant - The locus participant object.
```

Reply:
@edvujic I did the same as for other docs, don't want to do only here different style, can change everywhere as an option